examples/chip-tool/templates/tests.js

```diff
@@ -279,6 +284,7 @@ function getTests()
     Others, //
     SoftwareDiagnostics, //
     Subscriptions, //
+    Groups,
```

Comment:
```suggestion
    Groups, //
```
to keep the reformatter at bay